cds_ils/importer/providers/cds/rules/base.py

```diff
@@ -177,14 +177,21 @@ def special_serials(self, key, value):
 def document_type(self, key, value):
     """Translates document type field."""
     _doc_type = self.get("document_type", {})
+    _migration = self["_migration"]
 
     def doc_type_mapping(val):
         if val:
             return mapping(DOCUMENT_TYPE, val)
 
     for v in force_list(value):
-        val_a = doc_type_mapping(clean_val("a", v, str))
-        val_b = doc_type_mapping(clean_val("b", v, str))
+        value_a = clean_val("a", v, str)
+        value_b = clean_val("b", v, str)
+        if key == "690C_" and (
```

Comment:
if I remember correctly, if 690C will be equal YELLOW REPORT then in line 193 it will fail the migration anyway, since yellow report is not in the mapping, please correct me if I am wrong

Reply:
It will throw `ERROR: [UNEXPECTED INPUT VALUE]` and the record is migrated anyway, at least locally (this is one[ record example](https://cds.cern.ch/record/923393))

Reply:
yes, but from the code you wrote it means that value it is expected, so it is confusing. If you take a look on the two previous functions, you are already processing 690C_ there. Keeping 3 different functions for one MARC field is a little confusing. I can explain in detail in a call